idr0053-study.txt

```diff
@@ -0,0 +1,93 @@
+# STUDY DESCRIPTION SECTION				
+				
+Comment[IDR Study Accession]	idr0053			
+Study Title	Virtual nanoscopy: Generation of ultra-large high resolution electron microscopy maps			
+Study Type	high resolution electron microscopy			
+Study Type Term Source REF				
+Study Type Term Accession				
+Study Description	A key obstacle in uncovering the orchestration between molecular and cellular events is the vastly different length scales on which they occur. We describe here a methodology for ultrastructurally mapping regions of cells and tissue as large as 1 mm(2) at nanometer resolution. Our approach employs standard transmission electron microscopy, rapid automated data collection, and stitching to create large virtual slides. It greatly facilitates correlative light-electron microscopy studies to relate structure and function and provides a genuine representation of ultrastructural events. The method is scalable as illustrated by slides up to 281 gigapixels in size. Here, we applied virtual nanoscopy in a correlative light-electron microscopy study to address the role of the endothelial glycocalyx in protein leakage over the glomerular filtration barrier, in an immunogold labeling study of internalization of oncolytic reovirus in human dendritic cells, in a cryo-electron microscopy study of intact vitrified mouse embryonic cells, and in an ultrastructural mapping of a complete zebrafish embryo slice.		 	
+Study Organism	Danio rerio, Mus musculus, Homo sapiens			
```

Comment:
This is the first time we have a multiple organism study right? Should these (and the term source REF/Accession lines below) be separate by tabs?
In terms of project annotation, I assume we will have 3 K/V pairs with the same key

Reply:
I think so, couldn't find any previous study which had multiple organisms. You're right, should be separated by tabs in that case. Hope duplicate keys won't be a problem for the project level map annotations.